plugins/events-manager/forms/event-editor.php

```diff
@@ -92,7 +92,7 @@
 				<div class="event-creator__three-up">
 					<div class="half">
 						<label class="event-form-details event-creator__label" for="event-description"><?php print __('Event description *', 'commuity-portal'); ?></label>
-						<textarea name="content" id="event-description" placeholder="<?php  print __('Add in the details of your event’s agenda here. If this is a multi-day event, you can add in the details of each day’s schedule and start/end time.', 'community-portal') ?>" rows="10" id="event-description" class="event-creator__input event-creator__textarea" style="width:100%" required maxlength="3000"><?php echo __($EM_Event->post_content) ?></textarea>
+						<textarea name="content" id="event-description" placeholder="Add in the details of your event’s agenda here. If this is a multi-day event, you can add in the details of each day’s schedule and start/end time." rows="10" id="event-description" class="event-creator__input event-creator__textarea" style="width:100%" required maxlength="3000"><?php echo __($EM_Event->post_content) ?></textarea>
```

Comment:
Placeholder needs to be wrapped here

Reply:
Also post_content doesn't need to be wrapped

Reply:
Sorry! Misread the first comment thought it was referring to the placeholder. This has been switched back